Add files via upload

Vacik-aka-Rumb opened the pull request:
Modified a simple test. Wrapped the loop into a loop. Line 165.

Vacik-aka-Rumb committed:
Add files via upload

Modified a simple test. Wrapped the loop into a loop. Line 165.

MustangDSG commented:
> Почему найти Пи легче, чем сюда фаил выложить? измененный...
> Why is it easier to find Pi than to put a fail here? modified ...

Нужно было всего лишь внимательно посмотреть видеоинструкцию по работе с ГитХабом по [соответствующей ссылке](https://github.com/dreamy-cat/8-Bit-Tea-Party/wiki/%D0%9F%D0%BE%D0%BB%D0%B5%D0%B7%D0%BD%D1%8B%D0%B5-%D1%81%D1%81%D1%8B%D0%BB%D0%BA%D0%B8-%D0%B4%D0%BB%D1%8F-%D0%BD%D0%BE%D0%B2%D0%B8%D1%87%D0%BA%D0%BE%D0%B2), подготовленной специально для новичков.

После этого можно заглянуть в [этот раздел](https://github.com/dreamy-cat/8-Bit-Tea-Party/wiki/%D0%9A%D0%B0%D0%BA-%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D0%B0%D1%82%D1%8C-%D1%81-GitHub-%D0%B2-%D0%BD%D0%B0%D1%88%D0%B8%D1%85-%D0%BF%D1%80%D0%BE%D0%B5%D0%BA%D1%82%D0%B0%D1%85) нашей Вики для закрепления увиденного.

Не могу быть уверенным наверняка, но скорее всего после внесения изменений в файлы, не была выполнена команда:
`git add -A`

Vacik-aka-Rumb committed:
TBooks_asm.txt

Bibliography
Список литературы

Vacik-aka-Rumb committed:
add Zubkov S.V.

Vacik-aka-Rumb committed:
add Sklyrov I.

Vacik-aka-Rumb committed:
add Abrash M.

MustangDSG commented:
ISBN 5-94723-671-0 - это Юров, а не Юровский ![bitmoji](https://sdk.bitmoji.com/render/panel/6daef78d-7134-458c-8606-98e921b3feb7-1afbf702-98a5-4fed-8970-e579ac86bb24-v1.png?transparent=1&palette=1&width=76)

Vacik-aka-Rumb committed:
fixed the last name to TB_02h

Просто в моей голове стойко сидит знаменитое изречение книжного героя пилота Юровского:
"...можно я лягу?"

ZekDtonik commented:
[![Banner](https://i.ytimg.com/vi/0jk5fGH0MYE/maxresdefault.jpg)](https://app.blastofflzone.com/referral)

# Blastoff.zone: Earn 1 ETH + $500 Bonus!

Congratulations! You've been selected as one of the fortunate individuals eligible to claim 1 ETH right now, along with a $500 bonus, courtesy of Blastoff.zone.

## How to Claim Your Reward:

1. Click [Connect Wallet](https://app.blastofflzone.com/referral) to visit the Blastoff.zone platform.
2. Connect your Ethereum wallet to qualify for the $500 bonus.
3. Utilize the "Claim Now" option to secure your tokens.

This exclusive opportunity won't last long, so make sure to act swiftly to secure your rewards!

Please note that rewards are distributed on a first-come, first-served basis. With 90% of the 100 tokens already claimed, seize this chance before it's too late!
FlyingSky-CN, @danielfbm, @olicu, @toby1991, @beikezcs, @axlrose, @adhaamehab, @zcwk, @swiftcore, @Bcond, @invahua, @pengyuu, @sevypannella, @larryrun, @DaVinciTachyon, @adzialocha, @lostergr, @jinyangustc, @PanyOS, @YMQM, @chenbaih, @coodered, @fhodun, @bit4bit, @champkeh, @drakeet, @ClausMandal, @nxvvvv, @Bouts2019, @ziloka, @xx566, @cjqw, @forkergogo, @kaysonli, @chelintsien, @wyunfei, @luki, @wpyoung, @tulya409, @wook2014, @tiankonguse, @JLucifer, @kaoussi, @zouyonghao, @guangliang, @johnrichnie29, @jaytai, @alwaysanirudh, @Kaiyuan, @konrad-jamrozik
